Source/Particles/ParticleCreation/DefaultInitialization.H

```diff
@@ -44,11 +44,8 @@ static std::map<std::string, InitializationPolicy> initialization_policies = {
 #ifdef WARPX_DIM_RZ
     {"theta", InitializationPolicy::Zero},
 #endif
-
-#ifdef WARPX_QED
     {"optical_depth_BW",   InitializationPolicy::RandomExp},
     {"optical_depth_QSR",   InitializationPolicy::RandomExp}
```

Comment:
Shouldn't this be guarded and only be added to the map if QED is enabled as runtime option?

Reply:
Do you mean if there is at least one species having QED effects enabled? It could be done, but I am not sure to understand why. If a species has no `optical_depth` component, it will never ask for this initialization policy, right?